tests/integration/targets/hcloud_image_info/tasks/prepare.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create test_server
+  hetzner.hcloud.hcloud_server:
+    name: "{{ hcloud_server_name }}"
+    server_type: cx11
+    image: ubuntu-22.04
+    state: stopped
+  register: test_server
+
+- name: Create test_snapshot
+  ansible.builtin.script:
+    cmd: >
+      {{ hcloud_cli_path }} server create-image
+      --type snapshot
+      --description "{{ hcloud_snapshot_name }}"
+      --label key=value
+      "{{ test_server.hcloud_server.id }}"
+      | awk '{print $2}'
```

Comment:
This uses `hcloud` CLI because the functionality is not supported in the ansible collection yet, right? #85

Reply:
Yes, this is one of the features that are lacking in the collection.

I tried to implement the new module, but I stopped as it would have been a rushed implementation.